packages/server/migrations/20241011165653_cascade-delete-grant-notes.js

```diff
@@ -0,0 +1,44 @@
+const { onUpdateTrigger } = require('../knexfile');
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+    const ON_UPDATE_TIMESTAMP_FUNCTION = `
+        CREATE OR REPLACE FUNCTION on_update_timestamp()
```

Comment:
Just making it a bit easier to locate quickly during a `psql` session :) 
```suggestion
        CREATE OR REPLACE FUNCTION before_update_set_updated_at()
```